ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -95,7 +90,7 @@ public QueuedCommandStatus enqueueCommand(
       final Statement statement,
       final KsqlConfig ksqlConfig,
       final Map<String, Object> overwriteProperties) {
-    final CommandId commandId = commandIdAssigner.getCommandId(statement);
+    final CommandId commandId = CommandIdAssigner.getCommandId(statement);
```

Comment:
Can we stick with a injected dependency, rather than a static function call, please? It makes types easier to test.  Statics end up coupling unit tests of one class with the implementation of other classes. (in this case you're coupling the `CommandStoreTest` with the implementation of `CommandIdAssigner`).